src/share/poudriere/common.sh

```diff
@@ -7060,8 +7060,13 @@ load_moved() {
 	[ -f ${MASTERMNT}${PORTSDIR}/MOVED ] || return 0
 	msg "Loading MOVED for ${MASTERMNT}${PORTSDIR}"
 	bset status "loading_moved:"
-	awk -f ${AWKPREFIX}/parse_MOVED.awk \
-	    ${MASTERMNT}${PORTSDIR}/MOVED | \
+        { cat ${MASTERMNT}${PORTSDIR}/MOVED 
```

Comment:
We can avoid calling `cat` with a variable. And should fixup quoting since touching this.
```
local movedfiles

movedfiles="${MASTERMNT}${PORTSDIR}/MOVED"
for o in ${OVERLAYS}; do
                 test -f "${MASTERMNT}${OVERLAYSDIR}/${o}/MOVED" || continue
                 movedfiles="${movedfiles} ${MASTERMNT}${OVERLAYSDIR}/${o}/MOVED"
done
awk -f ${AWKPREFIX}/parse_MOVED.awk ${movedfiles} | \
...
```

I doubt we would hit the command line argument limit here but if we do then this pattern would be more efficient
```
{
echo file1
for o; do
echo $o
done
} | xargs cat | awk -f ...
```

Best to avoid fork/exec if possible.